docs/application/web/guides/TOC.md

```diff
@@ -1,212 +1,209 @@
-## [Human Activity Monitor](sensors/ham.md)
-## [Device Sensors](sensors/device-sensors.md)
+## [Overview](/application/web/guides/sensors/sensors.md)
+## [Human Activity Monitor](/application/web/guides/sensors/ham.md)
+## [Device Sensors](/application/web/guides/sensors/device-sensors.md)
 
 # Text Input
-## [Overview](text-input/text-input.md)
-## [IME Application](text-input/input-method.md)
-## [Input Device](text-input/input-device.md)
+## [Overview](/application/web/guides/text-input/text-input.md)
+## [IME Application](/application/web/guides/text-input/input-method.md)
+## [Input Device](/application/web/guides/text-input/input-device.md)
 
 # Personal Data
-## [Overview](personal/personal.md)
-## [Account Management](personal/account.md)
-## [Bookmarks](personal/bookmarks.md)
-## [Calendar](personal/calendar.md)
-## [Call History](personal/call-history.md)
-## [Contacts](personal/contacts.md)
-## [Data Synchronization](personal/sync.md)
+## [Overview](/application/web/guides/personal/personal.md)
+## [Account Management](/application/web/guides/personal/account.md)
+## [Bookmarks](/application/web/guides/personal/bookmarks.md)
+## [Calendar](/application/web/guides/personal/calendar.md)
+## [Call History](/application/web/guides/personal/call-history.md)
+## [Contacts](/application/web/guides/personal/contacts.md)
+## [Data Synchronization](/application/web/guides/personal/sync.md)
 
 # Data Storage and Management
-## [Overview](data/data.md)
-## [Data Filtering and Sorting](data/data-filter.md)
-## [File System](data/file-system.md)
-## [Stored Content Management](data/stored-content.md)
-## [File Archiving](data/file-archiving.md)
+## [Overview](/application/web/guides/data/data.md)
+## [Data Filtering and Sorting](/application/web/guides/data/data-filter.md)
+## [File System](/application/web/guides/data/file-system.md)
+## [Stored Content Management](/application/web/guides/data/stored-content.md)
+## [File Archiving](/application/web/guides/data/file-archiving.md)
 
 # Device Settings and Systems
-## [Overview](device/device.md)
-## [Power States](device/power.md)
-## [System Information](device/system-information.md)
-## [System Settings](device/system-setting.md)
-## [Time and Date Management](device/time.md)
-## [Web View Settings](device/web-view.md)
-## [Sound and Vibration Feedback](device/feedback.md)
+## [Overview](/application/web/guides/device/device.md)
+## [Power States](/application/web/guides/device/power.md)
+## [System Information](/application/web/guides/device/system-information.md)
+## [System Settings](/application/web/guides/device/system-setting.md)
+## [Time and Date Management](/application/web/guides/device/time.md)
+## [Web View Settings](/application/web/guides/device/web-view.md)
+## [Sound and Vibration Feedback](/application/web/guides/device/feedback.md)
 
 # Security
-## [Secure Key Management](security/secure-key.md)
+## [Secure Key Management](/application/web/guides/security/secure-key.md)
 
```

Comment:
Question : We also have libteec.md, ppm.md and security.md files here but they are not listed here. I'm not sure if it's intended.

Reply:
Added:
[TEE Communication](/application/web/guides/security/tee-communication.md)
[Privacy-related Permissions](/application/web/guides/security/privacy-related-permissions.md)